### ISSUE TITLE
Questions for training codes

### ISSUE THREAD

**Issue by SeanChenxy** [otaheri/GrabNet#8]
Hi, Omid. Thanks for your sharing. I have two questions when I reading codes.
1) During training, why does NOT the `self.rhm_train`  need grad?  https://github.com/otaheri/GrabNet/blob/0b428a72d9575e196503ec56b4c4f5b4fde65914/grabnet/train/trainer.py#L78
2) What is the difference between `fpose_rhand_rotmat_f` and `fpose_rhand_rotmat` in training data?
Hope for your reply. Thanks.

**Comment by otaheri** [comment]
Hi @SeanChenxy 

Thank you for your interest in our work.

1. This is a good question. One can also set the self.rhm_train model parameters with require_grad but I think it will have no effect on the training. During the model (GrabNet) training, the MANO parameters are generated using GrabNet which means they will require_grad enabled in any case (and they replace the initial MANO parameters that we define). So, I assume changing it to require_grad would not break anything.

2. The detailed instruction on what is included in the dataset will be added to the website and repo soon. The ground_truth hand parameters are the ones without any extension at the end like  ```fpose_rhand_rotmat```. The ```_f``` at the end stands for "fake" and shows the parameters that we used as input when training RefineNet. In other words, the parameters with ```_f``` at the end, are the ground_truth parameters when we added random noise to them.

I hope these answer your questions.
